ui_framework/components/bar/bar.js

```diff
@@ -0,0 +1,13 @@
+import React, {
+  PropTypes,
+} from 'react';
```

Comment:
PropTypes should be imported directly, ala:
`import PropTypes from 'prop-types';`
https://facebook.github.io/react/warnings/dont-call-proptypes.html